terraform/aws/aws.tf

```diff
@@ -164,6 +165,56 @@ resource "aws_volume_attachment" "mi-worker-nodes-lvm-attachment" {
   force_detach = true
 }
 
+resource "aws_ebs_volume" "mi-kube-worker-lvm" {
```

Comment:
we're keeping the nomenclature consistent within these files because changing it would break existing deployments. We'll probably have a breaking change sometime soon anyway, and we'll take the opportunity to do this then.